networks/grpc/gRPC_test.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2019 The klaytn Authors
+// This file is part of the klaytn library.
+//
+// The klaytn library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+package grpc
+
+import (
+	"encoding/json"
+	"github.com/klaytn/klaytn/networks/rpc"
+	"github.com/stretchr/testify/assert"
+	"sync"
+	"testing"
+	"time"
+)
+
+const (
+	TEST_BLOCK_NUMBER = float64(123456789)
+)
+
+type APIgRPC struct{}
+
+func (a APIgRPC) BlockNumber() float64 {
+	return TEST_BLOCK_NUMBER
+}
+
+func TestGRPC(t *testing.T) {
+	waitGroup := &sync.WaitGroup{}
```

Comment:
How about defining a global variable for the wg?

Reply:
Okay sure. Why not?

Reply:
This a minor comment but I mean declaring a global variable so that it does not need to be passed as a function parameter.

Reply:
It may be better to avoid using global variables. Wouldn't it be better to only know the functions that need WaitGroup?